src/main/java/com/wolox/training/models/User.java

```diff
@@ -0,0 +1,84 @@
+package com.wolox.training.models;
+
+import com.wolox.training.exceptions.books.BookAlreadyOwnedException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+  @Id
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "USER_SEQ")
+  @SequenceGenerator(name = "USER_SEQ", sequenceName = "USER_SEQ")
+  private Long id;
+
+  @Column(nullable = false, unique = true)
+  private String userName;
+
+  @Column(nullable = false)
+  private String name;
+
+  @Column(nullable = false)
+  private LocalDate birthDate;
+
+  @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, orphanRemoval = true)
```

Comment:
Two things to comment here: 

1) We don't really need orphanRemoval, it is used when you want to delete an orphan object. But in this case we want that the user could still exists even if there are no books attached to it (And viceversa)

2) This is not the correct CascadeType. By saying ALL, you are saying: Persist, Merge, Refresh, Delete and Detach.

Detach is only so that something gets detached along the cascade when you detach an object from the persistence context manually. This almost never happens, so we won't focus on this point.

Persist says: If you persist the Users instance to the DB, then persist every item that this collection has in the DB as well.
Delete says: If you delete the Users instance from the DB, then delete every item that this collection has in the DB as well.
Merge says: If you changed the in-memory values of the collection, then merge this changes to the DB so these are reflected there.
Refresh says: If something changed in the database values of the collection, then refresh those changes in the in-memory values.

[Here](https://www.baeldung.com/jpa-cascade-types) is a link explaining it. So, you can choose what you really need from there :)